test_rclcpp/test/test_subscription.cpp

```diff
@@ -22,7 +22,7 @@
 #include <test_rclcpp/msg/u_int32.hpp>
 
 #ifdef RMW_IMPLEMENTATION
-# define CLASSNAME_(NAME, SUFFIX) NAME ## __ ## SUFFIX
+# define CLASSNAME_(NAME, SUFFIX) NAME##__##SUFFIX
```

Comment:
Please don't change this. You likely don't have an up-to-date version of the ament linter repo.